config/api.yml

```diff
@@ -2046,6 +2027,25 @@
         :identifier: vm_protect
       - :name: resolve
         :identifier: vm_policy_sim
+    :snapshots_subcollection_actions:
+      :get:
+      - :name: read
+        :identifier: vm_snapshot_view
+      :post:
+      - :name: create
+        :identifier: vm_snapshot_add
+      - :name: delete
+        :identifier: vm_snapshot_delete
+    :snapshots_subresource_actions:
+      :get:
+      - :name: read
+        :identifier: vm_snapshot_view
+      :post:
+      - :name: delete
+        :identifier: vm_snapshot_delete
+      :delete:
+      - :name: delete
+        :identifier: vm_snapshot_delete
```

Comment:
Can you double check that spec/requests/api/config_spec.rb's test "contains only valid miq_feature identifiers" also test subresource actions as you have above ?   If not, could you create another PR to update that test accordingly ?   Thanks 🎵